moc-health-check/moc_health_check.py

```diff
@@ -0,0 +1,271 @@
+                    b_raw                       = True,
+                    b_httpResponseBodyParse     = True,
+                    jsonwrapper                 = '',
+                    authToken                   = "password"
+        )
+        print(dataComs())
+        return dataComs()
+
+
+    # check status of job in Pman
+    def run_pman_status(self):
+        data = {   "action": "status",
+            "meta":  {
+                "key":"jid",
+                "value": '"' + self.JID + '"'
+            }
+        }
+
+        dataComs = pfurl.Pfurl(
+                    msg                         = json.dumps(data),
+                    verb                        = 'POST',
+                    http                        = '%s/%s' % (self.PMAN_URL, "api/v1/cmd"),
+                    b_quiet                     = False,
+                    b_raw                       = True,
+                    b_httpResponseBodyParse     = True,
+                    jsonwrapper                 = '',
+                    authToken                   = "password"
+        )
+        print(dataComs())
+
+        return dataComs()
+
+    # downlad data from swift
+    def run_pfioh_pull(self):
+        data = {"action": "pullPath",
+    "meta": {
+        "remote": {
+            "key":         '"' + self.JID + '"'
+        },
+        "local": {
+            "path":         "/tmp/" + self.SIZE,
+            "createDir":    True
+        },
+        "transport": {
+            "mechanism":    "compress",
+            "compress": {
+                "archive":  "zip",
+                "unpack":   True,
+                "cleanup":  True
+            }
+        }
+    }}
+
+        dataComs = pfurl.Pfurl(
+                    msg                         = json.dumps(data),
+                    verb                        = 'POST',
+                    http                        = '%s/%s' % (self.PFIOH_URL, "api/v1/cmd"),
+                    b_quiet                     = False,
+                    b_raw                       = True,
+                    b_httpResponseBodyParse     = True,
+                    jsonwrapper                 = '',
+                    authToken                   = "password"
+        )
+        print(dataComs())
+
+        return dataComs()
+    
+    # check if success rate is above threshold
+    def conditionals(self,threshold,success_pfioh_push,success_pfioh_pull,success_pman_status,success_pman_run):
+        state = True
+        msg = ""
+
+        if threshold > success_pfioh_push:
+            msg = ", Pfioh Push"
+            state = False
+        if threshold > success_pman_run:
+            msg += ", Pman Run"
+            state = False
+        if threshold > success_pman_status:
+            msg += ", Pman Status"
+            state = False
+        if threshold > success_pfioh_pull:
+            msg += ", Pfioh Pull"
+        return state, msg
+    
+    # create reentrancy by maintaining logs of program

```

Comment:
I thought this changed to not rely on the log?

Reply:
yes but I didn't delete the method since it may be useful in other scenarios. the method is present but it is not used in the program

Reply:
Let's not leave dead code unless we can think of a concrete case we are going to need it.